semconv/src/main/java/io/opentelemetry/semconv/SemanticAttributes.java

```diff
@@ -1,3910 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.semconv;
-
-import static io.opentelemetry.api.common.AttributeKey.booleanKey;
-import static io.opentelemetry.api.common.AttributeKey.doubleKey;
-import static io.opentelemetry.api.common.AttributeKey.longKey;
-import static io.opentelemetry.api.common.AttributeKey.stringArrayKey;
-import static io.opentelemetry.api.common.AttributeKey.stringKey;
-import static io.opentelemetry.semconv.AttributeKeyTemplate.stringArrayKeyTemplate;
-import static io.opentelemetry.semconv.AttributeKeyTemplate.stringKeyTemplate;
-
-import io.opentelemetry.api.common.AttributeKey;
-import java.util.List;
-
-/**
- * @deprecated This class is deprecated and will be removed in a future release. It is only provided
- *     as a convenience to help migration to the new semantic conventions classes structure that was
- *     introduced in version 1.24.0.
- */
-@Deprecated
-@SuppressWarnings("unused")
-public final class SemanticAttributes {
```

Comment:
We're keeping a copy of this and `ResourceAttributes` around to ease the transition.

Reply:
Fixed